### PR TITLE
GPII-2524: Updating dependencies and fixing up config and code

### PR DESCRIPTION
**Pull request by jobara**
https://issues.gpii.net/browse/GPII-2524

See: conversation in the fluid-work channel ( https://botbot.me/freenode/fluid-work/2017-08-10/?msg=89657787&page=1 )

### CODE REVIEW

**Comment by gtirloni** [comment]
Could you add these lines to the Vagrantfile just so we're on the same page with other repos? https://github.com/GPII/universal/blob/master/Vagrantfile#L33-L38

Other than that, it looks good! Thank you.

**Comment by jobara** [comment]
@gtirloni thanks added the vagrant config changes

**Comment by jobara** [comment]
@gtirloni do you have access to merge this PR, or should we ask @cindyli?

**Comment by gtirloni** [comment]
Unfortunately I don't have access.

**Comment by cindyli** [comment]
Running `node tests/js/firstDiscoveryServer-tests.js` fails with this error:

```
FATAL ERROR: Uncaught exception: Assertion failure - check console for more details: Error in assembleCreatorArguments: cannot look up component type name gpii.express.tests.caseHolder to a component creator grade with an argumentMap
```

**Comment by jobara** [comment]
@cindyli the test should all be passing now. I also updated to eslint. Ready for more review.

File: tests/js/firstDiscoveryServer-tests.js [hunk 1]
```diff
@@ -21,6 +22,8 @@ kettle.loadTestingSupport();
 
 require("../../src/js/firstDiscoveryServer.js");
 require("gpii-express/tests/js/lib/test-helpers.js");
```
Comment (cindyli) [inline tests/js/firstDiscoveryServer-tests.js]: This file seems no longer needed. Can you please double check? Thanks.

**Comment by jobara** [comment]
@cindyli i removed the require statement for test-helpers.js. Ready for more review.

**Comment by cindyli** [comment]
Merged at c8073ef39890ccdaa0207860419ec21e2b54ac27

**Pull request merged**